packages/rum-recorder/src/domain/rrweb-snapshot/snapshot.spec.ts

```diff
@@ -73,3 +73,52 @@ describe('absolute url to stylesheet', () => {
     expect(absoluteToStylesheet(`url('')`, href)).toEqual(`url('')`)
   })
 })
+
+describe('serializeNodeWithId', () => {
+  describe('ignores some nodes', () => {
+    const defaultOptions = {
+      doc: document,
+      skipChild: false,
+      map: {},
+    }
+
+    it('does not same ignored nodes in the map', () => {
```

Comment:
I think there is a typo here, should it be `does not save ignored nodes in the map` ?

Reply:
Good catch!